From randy.dunlap@oracle.com  Thu Oct 29 11:40:30 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Thu, 29 Oct 2009 09:52:02 -0700
Subject: Staging: fix wireless drivers depends
To: Stephen Rothwell <sfr@canb.auug.org.au>, gregkh@suse.de
Message-ID: <20091029095202.09161c1e.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

These drivers can (erroneously) be enabled even when
CONFIG_NET=n, CONFIG_NETDEVICES=n, CONFIG_WLAN=n, etc.
Stop this.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8187se/Kconfig |    2 +-
 drivers/staging/rtl8192e/Kconfig  |    2 +-
 drivers/staging/rtl8192su/Kconfig |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/rtl8187se/Kconfig
+++ b/drivers/staging/rtl8187se/Kconfig
@@ -1,6 +1,6 @@
 config RTL8187SE
 	tristate "RealTek RTL8187SE Wireless LAN NIC driver"
-	depends on PCI
+	depends on PCI && WLAN
 	depends on WIRELESS_EXT
 	default N
 	---help---
--- a/drivers/staging/rtl8192e/Kconfig
+++ b/drivers/staging/rtl8192e/Kconfig
@@ -1,6 +1,6 @@
 config RTL8192E
 	tristate "RealTek RTL8192E Wireless LAN NIC driver"
-	depends on PCI
+	depends on PCI && WLAN
 	depends on WIRELESS_EXT
 	default N
 	---help---
--- a/drivers/staging/rtl8192su/Kconfig
+++ b/drivers/staging/rtl8192su/Kconfig
@@ -1,6 +1,6 @@
 config RTL8192SU
 	tristate "RealTek RTL8192SU Wireless LAN NIC driver"
-	depends on PCI
+	depends on PCI && WLAN
 	depends on WIRELESS_EXT
 	default N
 	---help---
